fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/keyspace/KeySpacePathImpl.java

```diff
@@ -337,6 +337,32 @@ public String toString() {
         return sb.toString();
     }
 
+    @Override
+    public String toString(@Nonnull Tuple t) {
```

Comment:
It seems a little unfortunate to me that this at `toString(void)` are so similar in implementation. Should this take an `@Nullable` tuple and then `toString` becomes `toString(null)`?

Reply:
Good suggestion.